include/baresip.h

```diff
@@ -151,6 +151,20 @@ struct call  *call_find_linenum(const struct list *calls, uint32_t linenum);
 void call_set_current(struct list *calls, struct call *call);
 
 
+/*
+* Custom headers
+*/
+const struct list *call_get_custom_hdrs(const struct call *call);
```

Comment:
perhaps move this prototype to the `struct call` block since it is accessing `call` ?